Sources/StreamChatUI/Resources/en.lproj/Localizable.strings

```diff
@@ -52,6 +52,7 @@
 "composer.title.edit" = "Edit Message";
 "composer.title.reply" = "Reply to Message";
 "composer.placeholder.message" = "Send a message";
+"composer.placeholder.messageDisabled" = "You can't send messages in this channel";
```

Comment:
Is this something we have in Figma? Do you know how other SDKs handle it?

Reply:
Yes we have it in figma, and Android uses the same copy